config/environments/test.rb

```diff
@@ -43,4 +43,6 @@
   # config.assets.compile = true
   # config.assets.js_compressor = :uglifier
   config.active_job.queue_adapter = :sidekiq
+  # Configure the drafts strorage directory
+  config.drafts_storage_dir = '/tmp/drafts/'
```

Comment:
Do we want to put this in the system `/tmp/drafts/`, or in `./tmp/drafts/`. I'm okay with system if there's a reason, this is just a change it makes sense to discuss.